_templates/playground/new/react.md.ejs.t

```diff
@@ -11,9 +11,10 @@ import './main.css';<% } %>
 
 function Example() {
   return (
-    <<%= pascalComponent %>></<%= pascalComponent %>>
+    <>
+      <<%= pascalComponent %>></<%= pascalComponent %>>
```

Comment:
This change was made because running `npm run lint` after adding a new playground was removing the `()` but this makes it so it passes lint and supports adding more child components.